src/Tizen.NUI/src/public/BaseComponents/ControlState.cs

```diff
@@ -86,20 +85,25 @@ public class ControlState : IEquatable<ControlState>
         /// This is used in a selector class. It represents all other states except for states that are already defined in a selector.
         /// </summary>
         /// <since_tizen> 9 </since_tizen>
-        public static readonly ControlState Other = Create("Other");
+        public static readonly ControlState Other = new ControlState(nameof(Other));
+
+        readonly ulong bitFlags;
 
-        private List<ControlState> stateList = new List<ControlState>();
-        private readonly string name = "";
 
         /// <summary>
         /// Gets or sets a value indicating whether it has combined states.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public bool IsCombined => stateList.Count > 1;
+        public bool IsCombined => bitFlags != 0UL && Math.Ceiling(Math.Log2(bitFlags)) != Math.Floor(Math.Log2(bitFlags));
```

Comment:
```suggestion
        public bool IsCombined => bitFlags != 0UL && ((bitFlags & (bitFlags - 1UL) != 0UL);
```

I dont like float point operation.

As I understand, this bool used if `bitFlags` has two or more bits. If than, we can detect that case by simple bitwise operation.

Reply:
이야... 안그래도 지윤님이랑 뭔가 더 쉬운 방법 있지 않을까 했는데 역시!!